lib/new_relic/agent/instrumentation/grpc/server/instrumentation.rb

```diff
@@ -44,6 +48,12 @@ def run_with_tracing(*args)
 
           private
 
+          def process_distributed_tracing_headers(active_call)
+            return unless active_call.metadata
+
+            ::NewRelic::Agent::DistributedTracing::accept_distributed_trace_headers(active_call.metadata, 'Other')
```

Comment:
I'm not familiar with the syntax of using `::` before a method. Is this another way to call a class method? I noticed this is also used in our Sidekiq instrumentation.

Reply:
Oh neat, I hadn't noticed that. This syntax was copied from `lib/new_relic/agent/instrumentation/sidekiq.rb` so that's why they're aligned.

`DistributedTracing` is a module that extends `self`, so the `accept_distributed_trace_headers` can be directly invoked like that. But as far as I know, it's equivalent to just using a `.` with `DistributedTracing.accept_distributed_trace_headers`. I wonder if there's a RuboCop cop for preferring one style over the other.

Reply:
Taking a cursory look at the RuboCop docs didn't reveal such a cop to me. I searched using `class method`, `module`, `::`, and `call`. Perhaps there's an opportunity for a contribution!